dotnet6/web/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -7,6 +7,9 @@ Description: >
 Globals:
   Function:
     Timeout: 100
+  Api:
```

Comment:
This template contains "HttpApi" events but not "Api". So this is not applicable as well. Should remove it.